Cargo.lock

```diff
@@ -4257,9 +4257,9 @@ dependencies = [
 
 [[package]]
 name = "rustls"
-version = "0.21.11"
+version = "0.21.10"
```

Comment:
dependabot just recently bumped `0.21.10` --> `0.21.11` [in this PR](https://github.com/impierce/did-manager/pull/10) since it contained a security vulnerability. It didn't affect us, but we still shouldn't revert the version.